code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -597,40 +604,40 @@
 		return
 
 	// Can we increase the carp spawn pool size?
-	if(last_carp_inc >= 40)
+	if(last_carp_inc >= carp_interval)
 		carp_stored++
 		icon_state = "carp_rift_carpspawn"
 		if(light_color != LIGHT_COLOR_PURPLE)
 			set_light_color(LIGHT_COLOR_PURPLE)
 			update_light()
 		notify_ghosts("The carp rift can summon an additional carp!", source = src, action = NOTIFY_ORBIT, flashwindow = FALSE, header = "Carp Spawn Available")
-		last_carp_inc -= 40
+		last_carp_inc -= carp_interval
 
 	// Is the rift now fully charged?
 	if(time_charged >= max_charge)
 		charge_state = CHARGE_COMPLETED
 		var/area/A = get_area(src)
-		priority_announce("Spatial object has reached peak energy charge in [initial(A.name)], please stand-by.", "Central Command Spatial Corps")
+		priority_announce("Spatial object has reached peak energy charge in [initial(A.name)], please stand-by.", "Central Command Wildlife Observations")
```

Comment:
The announcement subject is still a "Spatial object" despite the title being about wildlife.